### PR TITLE
enable VM resource quota for Azure provider

### LINKED ISSUE #6472: Add resource quota support for Azure provider

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: enable VM resource quota for Azure provider

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #6472

```release-note
NONE
```

### CODE REVIEW

File: pkg/handler/common/provider/azure.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"net/http"
 
+	kubermaticv1 "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
```
Comment (lsviben) [inline pkg/handler/common/provider/azure.go]: NIT: put the import down to its group

**Review by lsviben — COMMENTED** [review]
Looks good, one minor comment

File: pkg/handler/common/provider/azure.go (hunk 1 above)
Comment (zreigz) [reply pkg/handler/common/provider/azure.go]: done

**Pull request merged**